osu.Game/Rulesets/Edit/DistancedHitObjectComposer.cs

```diff
@@ -273,7 +273,7 @@ public virtual float DurationToDistance(HitObject referenceObject, double durati
         public virtual double DistanceToDuration(HitObject referenceObject, float distance)
         {
             double beatLength = BeatSnapProvider.GetBeatLengthAtTime(referenceObject.StartTime);
-            return distance / GetBeatSnapDistanceAt(referenceObject) * beatLength;
+            return distance / GetBeatSnapDistanceAt(referenceObject) * beatLength * referenceObject.DifficultyControlPoint.SliderVelocity;
```

Comment:
This confuses me. `GetBeatSnapDistanceAt()` will now include the reference slider velocity by default here, but this line also adds a multiplication by `referenceObject.DifficultyControlPoint.SliderVelocity` - which as far as I can tell will undo the reference slider velocity inclusion (and be tantamount to calling `GetBeatSnapDistanceAt(..., false)`). However `DurationToDistance()` - that is supposed to be the inverse of this function - does not do any of this. What am I not seeing?

Reply:
Yeah something is wrong here. This line should not have been committed (it was my initial approach before adding the new parameter). Unfortunate that the new tests didn't catch this one.

Reply:
Fixed, and added test coverage of the two methods which were missing it...